mbf_abstract_nav/include/mbf_abstract_nav/abstract_controller_execution.h

```diff
@@ -220,11 +223,11 @@ namespace mbf_abstract_nav
     //! the name of the loaded plugin
     std::string plugin_name_;
 
-    //! the local planer to calculate the velocity command
+    //! The local planer to calculate the velocity command
     mbf_abstract_core::AbstractController::Ptr controller_;
 
-    //! shared pointer to the shared tf listener
-    const TFPtr &tf_listener_ptr;
+    //! Reference to the current robot state
+    mbf_utility::RobotInformation &robot_info_;
```

Comment:
It should be a const reference, but bloody OdometryHelper doesn't declare its methods as const
Options:
- PR OdometryHelper: will take ages to merge, if ever
- Copy it onto mbf_utility and modify
- Let it this way

If nothing against, I'l go for the 1st and 2nd options, on a separated commit for easy reverting, in case 1st gets merged fast.
Sounds good?  @spuetz , @dorezyuk

Reply:
A better solution is to declare the odom helper in the RobotInformation as mutable (and thus make the methods using the odom helper const)

Reply:
ah...  good idea!  changed

Reply:
coopted odom helper